components/layout/display_list/webrender_helpers.rs

```diff
@@ -233,7 +243,9 @@ impl WebRenderDisplayItemConverter for DisplayItem {
                     stacking_context.transform_style,
                     stacking_context.mix_blend_mode,
                     &stacking_context.filters,
+                    &[],
                     RasterSpace::Screen,
+                    /* cache_tiles = */ false,
```

Comment:
You should probably set this to true only for the root stacking context. @gw3583 can probably confirm that.

Reply:
I would probably just leave it as false unconditionally, for now. That will preserve existing behavior. Enabling picture caching could be done as a separate step / follow up.